theano/gof/graph.py

```diff
@@ -125,8 +125,12 @@ def run_params(self):
         Returns the params for the node, or NoParams if no params is set.
 
         """
-        if hasattr(self.op, 'get_params'):
-            return self.op.get_params(self)
+        if hasattr(self.op, 'params_type'):
```

Comment:
Ok, I didn't know that there was an old interface where get_params() is the only one expected attribute.

I have changed it because I add a default implementation for get_params() (which checks if op has a Wrapper as params_type), so that all ops have now a get_params by default.

I will use this code instead:
```python
def run_params(self):
    try:
        return self.op.get_params(self)
    except MethodNotDefined:
        # the default implementation of get_params() raises a 
        # MethodNotDefined exception if it does not find a Wrapper
        return NoParams
```